azurerm/data_source_route_table.go

```diff
@@ -17,7 +17,7 @@ func dataSourceArmRouteTable() *schema.Resource {
 			"name": {
 				Type:         schema.TypeString,
 				Required:     true,
-				ValidateFunc: validation.NoZeroValues,
+				ValidateFunc: validate.NoEmptyStrings(),
```

Comment:
We shouldn't need to call the function:
```suggestion
				ValidateFunc: validate.NoEmptyStrings,
```